missing imports to go along with exports

wheaties opened the pull request:
Looks like i missed the ```@imports``` to go along with those 3 new export types.

milessabin commented:
Blimey ... I should have spotted that those were missing. Is there a test you could add which would show that these are necessary?

wheaties commented:
I moved the set of tests I had added to "exporttests.scala" into "typedefns.scala" and added `@imports` annotations. Without the code, the imports would not have compiled. I hesitated to add code that would add testing of import functionality as I didn't see a straight forward way of recreating what you had in the other files.

milessabin commented:
OK ... LGTM ... Many thanks ... merging :-)
